client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -863,9 +872,12 @@ export default {
       }
 
       this.setSegment({ ...updated, id: this.segment.id })
+
+      return relations
     },
 
     updateSegment (key, val) {
+      console.log('updateSegment', key, val)
```

Comment:
This can be removed I guess

Reply:
0c1aec2fd9c492da1f7e43eee516588b545fafca